extern/sector-storage/ffiwrapper/sealer_test.go

```diff
@@ -869,89 +868,6 @@ func TestAddPiece512MPadded(t *testing.T) {
 	require.Equal(t, "baga6ea4seaqonenxyku4o7hr5xkzbqsceipf6xgli3on54beqbk6k246sbooobq", c.PieceCID.String())
 }
 
-func setupLogger(t *testing.T) *bytes.Buffer {
```

Comment:
Could you explain why this needs to go entirely? I understand that `generated` no longer exists, but has all this functionality vanished from the library? Or is it just behind different APIs? If the latter, we should adapt the tests, not drop the coverage.

Reply:
its gone completely afaict.

Reply:
It isn't. AFAIK no functionality has been removed, only APIs have changed. Here's where the log is now:

filecoin-project/filecoin-ffi@bb281b5/cgo/util.go#L11

Reply:
ok feel free to reinstate, it is really ugly however.

Reply:
Agreed that the form of the test is terrible, but it is providing certain coverage that we can't drop here.